packages/cogify/src/cogify/cli/cli.config.ts

```diff
@@ -51,13 +51,18 @@ export const BasemapsCogifyConfigCommand = command({
 
     const configPath = base58.encode(Buffer.from(outputPath));
     const location = getImageryCenterZoom(im);
-    const locationHash = `#@${location.lat.toFixed(7)},${location.lon.toFixed(7)},z${location.zoom}`;
+    const lat = location.lat.toFixed(7);
+    const lon = location.lon.toFixed(7);
+    const locationHash = `#@${lat},${lon},z${location.zoom}`;
     const url = `https://basemaps.linz.govt.nz/?config=${configPath}&i=${im.name}&tileMatrix=${im.tileMatrix}&debug${locationHash}`;
+    const preview = `https://basemaps.linz.govt.nz/v1/preview/${im.name}/${im.tileMatrix}/${location.zoom}/${lon}/${lat}.webp?config=${configPath}`;
```

Comment:
I wonder why do we want to change this as preview, don't we want a basemaps link that we can browse around?

Reply:
It spits out both, but means the slack alerts can come with a image until we add the direct link support

Reply:
I see, this is only for the logs.

Reply:
Its also a additive change, so the old URL is still there.